search/Sta.cc

```diff
@@ -366,7 +366,8 @@ Sta::updateComponentsState()
 void
 Sta::makeReport()
 {
-  report_ = new ReportTcl();
```

Comment:
// Default constructors that are called by makeComponents in the Sta
 The intended way to handle this is with a derived class like the comment in Sta.hh indicates
 // constructor.  These can be redefined by a derived class to
  // specialize the sta components.
  virtual void makeReport();

Reply:
Ah, ok, make sense, thank you